packages/core/ios/RNSentryVersion.m

```diff
@@ -1,5 +1,6 @@
 #import "RNSentryVersion.h"
 
 NSString *const NATIVE_SDK_NAME = @"sentry.cocoa.react-native";
+NSString *const REACT_NATIVE_SDK_NAME = @"sentry.javascript.react-native";
```

Comment:
Adding the new `RNSentryVersion.h` should fix the build issues.

Reply:
but its there 🤔

Reply:
ow, I just noticed its not here on GH. 🤔

This is the file in my machine
![image](https://github.com/user-attachments/assets/2724478a-6528-45ae-9bd3-267550f8a23a)

Reply:
I've done the same quite a few times with my commits 😓